### ISSUE TITLE
No translations for default themes

### ISSUE THREAD

**Issue by dftd** [mindstellar/i10n-osclass#15]
There's no files for default backend theme (Modern) and default frontend theme (Bender).
Atleast backend theme translations should be added imo

**Comment by navjottomer** [comment]
Backend theme translations are in core.po and message.po, theme.po (bender translations) is also included in every locale except a few. I carried away with backend theme changes.
This repo is for the next release of osclass, A workflow is still not complete to manage all translations automatically, I'll hope it will be complete soon.
Some languages are not complete, I still have to work on them, I can give you the access if you're interested.
See status here: https://crowdin.com/project/osclass-v2

**Comment by dftd** [comment]
Ah okay. I noticed when I translated everything that half of oc-admin is still in English. So backend Modern theme part is missing...

Anyways, I don't plan on doing translations for other languages, just wanted to freshen up the translations for my site.

BTW, there is a lot of duplicates or very similar sentences, some say items, some say listings, some have dots at the end, some not...
Before you start with properly translating, maybe fix that up in the core.

**Comment by navjottomer** [comment]
Yup, I too noticed those duplicates, those are on my to-do list. I'll keep this issue open for now.
